pagopa/pn-frontend
# Analog sending #pn 2985

## Short description
Analog sending in notification detail timeline. Introduced two new steps: SEND_ANALOG_PROGRESS and SEND_ANALOG_FEEDBACK managed as their digital analogues

## List of changes proposed in this pull request
- Updated model in NotificationDetail.ts
- Added two new classes in TimelineStepFactory.ts
- Added two new steps in TimelineAllowedStatus in notification.utility.ts
- Added legal fact management for these two new steps in notification.utility.ts
- Added localization in pa and pf

## How to test
Login in with marcopolo -> comune sappada and search notification with iun XVZY-RUEM-MQVP-202301-N-1 -> check that in timeline there are the analog steps.
This notification should contain the OK case.
